docs/tutorial-module/develop.rst

```diff
@@ -254,31 +247,73 @@ On the ``utils.py`` module will create a helper function that creates a record.
         # create uuid
         rec_uuid = uuid.uuid4()
         # add the schema
-        host = current_app.config.get('JSONSCHEMAS_HOST')
         data["$schema"] = \
             current_app.extensions['invenio-jsonschemas'].path_to_url(
-            'custom_record/custom-record-v1.0.0.json')
+                'records/custom-record-v1.0.0.json'
+            )
         # create PID
-        current_pidstore.minters['custid'](
-          rec_uuid, data, pid_value='custom_pid_{}'.format(rec_uuid)
-        )
+        current_pidstore.minters['recid'](rec_uuid, data)
         # create record
         created_record = Record.create(data, id_=rec_uuid)
         db.session.commit()
 
         # index the record
         indexer.index(created_record)
 
+5. Create the custom-record JSON Schema
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As you can see, our records use a custom schema. To define and use this schema,
+we need to write the ``custom-record-v1.0.0.json`` inside the ``records``
+folder of your data model project (``my-datamodel`` from the Quickstart
+tutorial :ref:`customize`).
+
+In ``my-datamodel/my-datamodel/jsonschemas/records/custom-record-v1.0.0.json``
+
+.. code-block:: json
+
+    {
+      "$schema": "http://json-schema.org/draft-04/schema#",
+      "id": "http://localhost/schemas/records/custom-record-v1.0.0.json",
+      "additionalProperties": true,
+      "title": "my-datamodel v1.0.0",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "Record title.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Record description.",
+          "type": "string"
+        },
+        "id": {
+          "description": "Invenio record identifier (integer).",
+          "type": "string"
+        }
+      },
+      "required": [
+        "title",
+        "description"
+      ]
+    }
+
+
 Demo time
 ---------
 
-Let's start our server again.
+Let's now see our Invenio module in action when integrated with our Invenio instance.
+We will install this module in our Invenio instance and then run the ``server``
+script. First go to the root folder of the ``invenio-unicorn`` module, then
```

Comment:
Shall we split this into two parts to make it more explicit? What do you think about the following:

>First we activate our instance virtualenv and we install the new `Invenio-Unicorn` module:
>```console
>$ workon my-repository-venv
>(my-repository-venv)$ pip install --editable .[all]
>```
>Then, we go to the instance folder, `my-repository`, and start the server:
>```console
>(my-repository-venv)$ ./scripts/server
>```
You can rephrase but I think splitting it would make it more clear!

Reply:
Yup, separating explanations in smaller chunks à-la Windows wizards works.